Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1.yml

```diff
@@ -0,0 +1,702 @@
+              right:
+                value:
+                  simple: FalconHost
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: state
+                iscontext: true
+              right:
+                value:
+                  simple: active
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 730,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "16":
+    id: "16"
+    taskid: d9d617d9-2efd-466e-8ce7-190f8db83b95
+    type: title
+    task:
+      id: d9d617d9-2efd-466e-8ce7-190f8db83b95
+      version: -1
+      name: McAfee ePolicy Orchestrator
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -650,
+          "y": 605
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "18":
+    id: "18"
+    taskid: 6cf08862-644d-479e-89ce-f9e173a8c562
+    type: title
+    task:
+      id: 6cf08862-644d-479e-89ce-f9e173a8c562
+      version: -1
+      name: Carbon Black Enterprise Response
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -180,
+          "y": 605
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "19":
+    id: "19"
+    taskid: 471d3862-a05c-42b1-871d-c1faa2fbb7a9
+    type: title
+    task:
+      id: 471d3862-a05c-42b1-871d-c1faa2fbb7a9
+      version: -1
+      name: Cylance Protect v2
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "26"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 270,
+          "y": 605
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "20":
+    id: "20"
+    taskid: 5d371f29-3a4c-43c5-8f71-b383db2e5320
+    type: title
+    task:
+      id: 5d371f29-3a4c-43c5-8f71-b383db2e5320
+      version: -1
+      name: CrowdStrike Falcon Host
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 730,
+          "y": 605
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "22":
+    id: "22"
+    taskid: 6a8160b7-fe32-4ab0-8396-e9bcafc98194
+    type: condition
+    task:
+      id: 6a8160b7-fe32-4ab0-8396-e9bcafc98194
+      version: -1
+      name: Is Active Directory Query v2 enabled?
+      description: Checks if there is any active instance of the Active Directory
```

Comment:
Checks if there is an active instance of the Active Directory Query v2 integration enabled.